.ci/deps.sh

```diff
@@ -105,8 +105,8 @@ npm install
 # R commands
 echo '.libPaths( c( "'"$R_LIB_USER"'", .libPaths()) )' >> .Rprofile
 echo 'options(repos=structure(c(CRAN="http://cran.rstudio.com")))' >> .Rprofile
-R -q -e 'install.packages("lintr")'
-R -q -e 'install.packages("formatR")'
+R -q -e 'install.packages("lintr")' 2>&1|grep -v '=\|downloaded'
```

Comment:
I dont think this is worthwhile. It isnt a percentage progress bar. It is only two additional lines per package, and these R pacakes are very noisy with configure and build phases.